src/main/resources/application-prod.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
```

Comment:
Isn't it necessary to add profiles here?

Reply:
no actually I have added spring profiles in application.properties file and this applicatio.prod, and i have mentioned which profile is active there, so i guess it is not necessary to add spring profiles here